.github/workflows/windows.yml

```diff
@@ -15,3 +15,38 @@ jobs:
         cd build
         cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_VERBOSE_MAKEFILE=ON -DAMReX_BUILD_TUTORIALS=ON -DAMReX_FORTRAN=OFF -DAMReX_MPI=OFF
         cmake --build . --config Debug
+
+  # Build libamrex and all tutorials with CUDA
+  tutorials-cuda:
+    name: MSVC C++17 w/ CUDA@11.1.1 w/o Fortran w/o MPI
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v2
+    - name: Windows CUDA Cache
+      uses: actions/cache@v2
+      with:
+        path: "C:/Program Files/NVIDIA GPU Computing Toolkit/"
```

Comment:
I hope this works. Dunno what Windows installs register these days in the system besides this.

Reply:
You might want the whole path to the bin directory `C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/vX.Y/bin/`

Reply:
But that's more specific, I don't want to miss auxiliary files of the install.